build.psm1

```diff
@@ -683,7 +683,7 @@ function Restore-PSPester
         [ValidateNotNullOrEmpty()]
         [string] $Destination = ([IO.Path]::Combine((Split-Path (Get-PSOptions -DefaultToNew).Output), "Modules"))
     )
-    Save-Module -Name Pester -Path $Destination -Repository PSGallery
+    Save-Module -Name Pester -Path $Destination -Repository PSGallery -MinimumVersion "4.2"
```

Comment:
We need to follow a process when we update a version.  Please use `-RequiredVersion`

Reply:
Fixed